README.md

```diff
@@ -44,6 +44,22 @@ To use this template:
    as your source. You can read more about generating your project
    in the [copier documentation](https://copier.readthedocs.io/en/stable/generating/).
 
+## How to run the test suite locally
```

Comment:
Friends, bare with me on the instructions. I'm still getting to know copier and am wrapping my head around the test suite works. it seems great as it caught some spacing issues in my workflow! I just don't fully understand it yet so if any of this isn't quite right please say the word!